pkg/signing/handlers/rsa/handler.go

```diff
@@ -148,7 +148,7 @@ func (h Handler) Verify(digest string, signature *signing.Signature, sctx signin
 	return nil
 }
 
-func (_ Handler) CreateKeyPair() (priv signutils.GenericPublicKey, pub signutils.GenericPublicKey, err error) {
+func (_ Handler) CreateKeyPair() (priv signutils.GenericPrivateKey, pub signutils.GenericPublicKey, err error) {
```

Comment:
down below the function seams also using the wrong parameter names - right? It's twice: `GenericPublicKey`

```golang
func CreateKeyPair() (priv signutils.GenericPublicKey, pub signutils.GenericPublicKey, err error) {
```

Reply:
Good catch, I adjusted it.